smoketest/scripts/cli/test_interfaces_bridge.py

```diff
@@ -25,14 +25,13 @@
 from vyos.ifconfig import Section
 from vyos.util import cmd
 from vyos.util import read_file
+from vyos.validate import is_intf_addr_assigned
 
 class BridgeInterfaceTest(BasicInterfaceTest.BaseTest):
     def setUp(self):
         self._test_ip = True
         self._test_ipv6 = True
         self._test_ipv6_pd = True
-        self._test_vlan = True
```

Comment:
Why remove the VLAN test?

Reply:
Well, the `vif` command is only effective when VLAN awareness is turned on, and the verification logic is strengthened when VLAN awareness is not turned on. Therefore, the VLAN verification of the bridge is special (it cannot be verified according to the VLAN verification method of the ordinary interface, otherwise the configuration error will be reported according to the normal logic) 

https://github.com/vyos/vyos-1x/pull/677/files/0f9f2753bc0d7d7884a2c57a4c8ed201c4a8347a#diff-7d2005786129d5a7a26624fb13ce3cb41deeb704ceb036e0243c28fbd7cd668cR171-R177

Reply:
Why not simply derive `test_8021q_vlan_interfaces()` and turn on the bridge for this testcase? We use the derive pattern already for some places.

Reply:
done